### PR TITLE
fix: added missing translations

### PR DESCRIPTION
**Pull request by kalpesh3000**
Added missing translations for the update-password.

### CODE REVIEW

File: .changeset/good-penguins-collect.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+"@saas-ui/auth": major
+"website": major
```
Comment (Pagebakers) [inline .changeset/good-penguins-collect.md:3]: This should be a patch change

File: .changeset/good-penguins-collect.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+"@saas-ui/auth": major
```
Comment (Pagebakers) [inline .changeset/good-penguins-collect.md:2]: Should be `patch`

**Review by Pagebakers — CHANGES_REQUESTED** [review]
Perfect, can you update the changeset to be a patch instead of major then i can merge it in and release it.

File: .changeset/good-penguins-collect.md (hunk 1 above)
Comment (kalpesh3000) [reply .changeset/good-penguins-collect.md]: Done.

File: .changeset/good-penguins-collect.md (hunk 2 above)
Comment (kalpesh3000) [reply .changeset/good-penguins-collect.md]: Done.

**Review by Pagebakers — APPROVED** [review]

**Pull request merged**